transform-test-selector-params-to-hash-pairs.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env node */
+var TEST_SELECTOR_PREFIX = /data-test-.*/;
```

Comment:
I guess the problem is that one uses ES6 modules and the other CommonJS, so reducing duplication there might not be quite as easy